Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -36,19 +36,17 @@ Seedfinder<external_spacepoint_t, platform_t>::Seedfinder(
 
 template <typename external_spacepoint_t, typename platform_t>
 template <typename sp_range_t>
-std::vector<Seed<external_spacepoint_t>>
-Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
+void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
+    State& state,
+    std::back_insert_iterator<std::vector<Seed<external_spacepoint_t>>> outIt,
```

Comment:
Is it necessary to template this on a concrete type? Wouldn't it be more flexible (and more efficient?) to allow the user to provide a small_vector with around the expected number of SP preallocated?

Reply:
I mean, the method is templated already, so in principle that should be possible I guess.

Reply:
Do you think that's overkill? You don't seem to be convinced :)

Reply:
Done.